### DISCUSSION TITLE
Calculating RSS on each receiver antenna element

### DISCUSSION THREAD

**Discussion by rajBakhunchhe** [NVlabs/sionna#713] [Q&A]
I have a 4*4 antenna array on receiver. i want to find out received signal strength on each antenna element (or on any specific antenna element), so that I can create a heat-map for the incoming direction of signals (spatial spectrum heatmap) with respect to receiver antenna. So far, I have only been able to find the CIR for each path, but not for each Rx antenna element. 
i am a beginner here (as well as in wireless communication) and I would really appreciate your help.

**Comment by SebastianCa** [comment]
Hi @rajBakhunchhe,
did you already have a look at the [RT tutorial](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html#)?
The [CIR](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Paths.cir) is of shape `[batch_size, num_rx, num_rx_ant, num_tx, num_tx_ant, max_num_paths, num_time_steps]`. You can loop over the `num_rx_ant` dimension to get the CIRs for each rx antenna element.
I hope this helps.
